environments/dev/group_vars/all.yml

```diff
@@ -84,23 +88,33 @@ init_scripts:
   hypothesis:
     template: var/www/h/bin/init_script
     dest: /var/www/h/bin/hypothesis
-    exec: python -m h "$@"
+    exec: exec python -m h "$@"
+    state: present
     env_vars: 
       <<: *default_env
   start_celery_worker:
     template: var/www/h/bin/init_script
     dest: /var/www/h/bin/start-celery-worker
-    exec: "python -m h celery worker --logfile=${CELERYD_LOG_FILE} --loglevel=${CELERYD_LOG_LEVEL}" 
+    exec: "exec python -m h celery worker --logfile=${CELERYD_LOG_FILE} --loglevel=${CELERYD_LOG_LEVEL}" 
```

Comment:
Originally the `init_script` template included the `exec` command which I removed.